tests/unit/backend/helpers/jest-db.ts

```diff
@@ -222,13 +230,45 @@ const getFullFormById = async (
 ): Promise<IPopulatedForm | null> =>
   await getEmailFormModel(mongoose).getFullFormById(formId)
 
+const insertVerifiedSms = async ({
+  formId,
+  formAdmin,
+  msgSrvcSid = 'MOCK MESSAGE SERVICE ID',
+  logType = LogType.success,
+  isOnboarded = false,
+}: {
+  formId: ObjectID
+  formAdmin: { email: string; userId: ObjectID }
+  msgSrvcSid?: string
+  logType?: LogType
+  isOnboarded?: boolean
+}): Promise<ISmsCountSchema> => {
+  const SmsModel = getSmsCountModel(mongoose)
+  const sms = await SmsModel.create({
+    smsType: SmsType.Verification,
+    logType,
+    msgSrvcSid,
+    form: formId,
+    formAdmin,
+    isOnboarded,
+  })
+
+  // Workaround because the isOnboarded property is set onSave.
```

Comment:
If put this function in the model tests directly, and mock `config`'s esrvcMsgId or something to be the `msgSrvcSid` we use to create, then we should get the default behaviour without needed this workaround.

Reply:
i removed this function because when we mock the behaviour of stuff 1 level below, we don't require this function anymore.